src/status_im/ui/components/icons/vector_icons.cljs

```diff
@@ -31,15 +157,15 @@
                            {:width  (or width 24)
                             :height (or height 24)})
      :accessibility-label accessibility-label}
-    [react/image {:source {:uri (keyword (clojure.core/name name))}
-                  :style  (cond-> {:width  (or width 24)
-                                   :height (or height 24)}
+    [react/image (merge {:style  (cond-> {:width  (or width 24)
+                                          :height (or height 24)}
 
-                            resize-mode
-                            (assoc :resize-mode resize-mode)
+                                   resize-mode
+                                   (assoc :resize-mode resize-mode)
 
-                            color
-                            (assoc :tint-color (match-color color)))}]]))
+                                   color
+                                   (assoc :tint-color (match-color color)))}
+                        (icon-source name))]]))
```

Comment:
no point making a merge here, just have :source key and make icon-source return the source as value directly in the map